### PR TITLE
Credit card fraud detection

### PR DESCRIPTION
**Pull request by sankalp-srivastava**
## Issue Title : Credit card fraud detection

-  Implement different algorithms like decision trees, logistic regression, and artificial neural networks to see which gives better accuracy. Compare the results of each algorithm and understand the behavior of models.

- [x] SWOC Participant 
- [ ] Contributor

Closes: #15 

### Describe the add-ons or changes you've made

added 3 .ipynb files that are EDA, baseline models and Bayesian optimization of Lightgbm algorithm. Updated readme in models folder and requirement.txt

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Ran the code from top to bottom on local machine as well as on kaggle.com and they generated no errors and ran succesfully.

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
1. Remove the `README.md` file from the Images folder.
2. Put the dataset in the Dataset folder and put the dataset link in the README.md file of the Dataset folder.
3. In the `requirements.txt` file, mention the versions of the libraries, like this,
```
numpy==<version>
pandas==<version>
...
...
...
```

@sankalp-srivastava do the needful changes please.

**Comment by sankalp-srivastava** [comment]
The following changes were made:
Dataset(.zip) is added
Removed readme from images folder
updated readme in dataset folder
updated requirements file with versions so it can be downloaded directly with pip

**Review by abhisheks008 — APPROVED** [review]

**Comment by abhisheks008** [comment]
@sankalp-srivastava congratulations on your first contribution 🎉
